src/Compatibility/Microsoft.DotNet.PackageValidation/BaselinePackageValidator.cs

```diff
@@ -40,7 +40,7 @@ public BaselinePackageValidator(Package baselinePackage, string noWarn, (string,
         /// <param name="package">Nuget Package that needs to be validated.</param>
         public void Validate(Package package)
         {
-            foreach (ContentItem baselineCompileTimeAsset in _baselinePackage.RefAssets)
```

Comment:
Why do you need this change?

Reply:
For most of the cases ref assets should work fine, but as we are moving away from having ref assets in package, i thought it might be best to replace it by compileTimeAssets here.

Reply:
On second thought adding it will decrease the performance as it will add duplicates in case of no ref assemblies in the package. reverted it back.